samples/subsys/power/stm32_pm/lowpower/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+config STM32L4_POWER_MGMT_EXAMPLE
+	bool
+	default y
+	select SYS_POWER_SLEEP_STATES
+	select HAS_SYS_POWER_STATE_SLEEP_1
```

Comment:
These are properties of SoC, so should be selected by the SoC entry.

Reply:
--> Soc Kconfig gets the  HAS_SYS_POWER_STATE_SLEEP_xxx
--> sample appli Kconfig gets the SYS_POWER_SLEEP_STATES and SYS_POWER_DEEP_SLEEP_STATES